tests/baselines/reference/transformApi/transformsCorrectly.transformAddDecoratedNode.js

```diff
@@ -11,6 +11,5 @@ var Foo = /** @class */ (function () {
     __decorate([
         Bar
     ], Foo.prototype, "baz", null);
-    Foo = __decorate([], Foo);

```

Comment:
Yes, it's because I changed [these lines](https://github.com/microsoft/TypeScript/pull/49089/files?show-viewed-files=true&file-filters%5B%5D=#diff-0d35c72f5d991babd4eb55d3d10704918654efe756e89025b60812238d405a78R341-R342) which previously passed an empty array for decorators (which we shouldn't have emit anything for in the first place).